xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -367,9 +367,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Asynchronously releases the unmanaged resources used by the <see cref="T:System.IO.Compression.BrotliStream" />.</summary>
+        <returns>A task that represents the asynchronous dispose operation.</returns>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+The <xref:System.IO.Compression.BrotliStream.DisposeAsync%2A> method enables you to perform a resource-intensive dispose operation without blocking the main thread. This performance consideration is particularly important in a [!INCLUDE[win8_appname_long](~/includes/win8-appname-long-md.md)] app or [!INCLUDE[desktop_appname](~/includes/desktop-appname-md.md)] app where a time-consuming stream operation can block the UI thread and make your app appear as if it is not working. The async methods are used in conjunction with the `async` and `await` keywords in Visual Basic and C#. 
```

Comment:
How do these show up on the site? `[!INCLUDE[win8_appname_long](~/includes/win8-appname-long-md.md)]`

Reply:
Like in Stream: https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.disposeasync?view=netcore-3.0

Reply:
Looks like plaintext to me. Why all the ceremony, then?
![image](https://user-images.githubusercontent.com/6527137/61560940-87796280-aa22-11e9-994d-f3ea0e5a83d8.png)

Reply:
These are include files. We've been using those less and less, but they're useful when the product name is still changing,

Reply:
It doesn't hurt to have them. Especially since all the DisposeAsync methods I've seen from other stream classes have the exact same remark message inherited from Stream.

Reply:
Sounds good to me.